test/tests_for_integration/api_to_db_test.py

```diff
@@ -199,6 +201,9 @@ def service(full_config, auth_url, mongo_client, config):
     # See the server file for the full scoop, but in short, the stop method expects a _proc
     # package variable to be set, but start doesn't always set it, and that causes an error.
 
+    # Tests are run in the same process to we need to be put the environment back the way it was
```

Comment:
`s/to/so/`

Reply:
I already have that fixed in the upcoming PR